Code_In_Progress/MyTextWebscraperV3.py

```diff
@@ -0,0 +1,188 @@
+from bs4 import BeautifulSoup
+import re
+from urllib.request import Request, urlopen
+import requests
+import time
+from nltk.corpus import stopwords
+
+import pandas as pd
+
+import re
+from nltk.stem import WordNetLemmatizer
+
+lemmatizer = WordNetLemmatizer()
+stop_words = stopwords.words('english')
+stop_words.extend(['http','org','com','www','lt','htm','html','https','youtu'])
+
+print(stop_words)
+file1 = open("Messages_pol.txt","a")
+file1.truncate(0)
+
+df = pd.DataFrame()
+df['cleaned_word'] = ""
+df['count_of_word_webscraper']=""
+df['ratio_of_use_webscraper']=""
+
+text=''
+
+
+
+def webscraper(Runtime):
+    newstring=''
+    new_output_list = []
+    output = ''
+    archive_column_numbers=[]
+    for i in range(Runtime):
+
+        print(i)
+        #if i==1:
+        #    i=2       ##I don't think i'll be needing this anymore
+        url = 'https://boards.4channel.org/pol/archive'
```

Comment:
per suggestions from the Report Out, you might want to grab an archive or two from a couple other relevant Reddit forums ("sus reddits") to create a more robust learning corpus